src/ResourceManager/Sql/Commands.Sql/TransparentDataEncryption/Cmdlet/AddAzureRmRmSqlManagedInstanceKeyVaultKey.cs

```diff
@@ -0,0 +1,66 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.Common;
+using Microsoft.Azure.Commands.Sql.Server.Model;
+using Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Adapter;
+using Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Security;
+
+
+namespace Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Cmdlet
+{
+    /// <summary>
+    /// Defines the Add-AzureSqlManagedInstanceKeyVaultKey cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.Add, "AzureRmSqlManagedInstanceKeyVaultKey", ConfirmImpact = ConfirmImpact.Low, SupportsShouldProcess = true)]
```

Comment:
Don't need ConfirmImpact, it should just be the default.  Apply everywhere (we should fix this for other cmdlets during the breaking change release, because this is unexpected for users).

Reply:
You also need a DefaultParameterSetName attribute